### ISSUE TITLE
Low sec storyline missions cause errors in console

### ISSUE THREAD

**Issue by Chryst** [ISeeDEDPpl/Questor#208]
Low sec storyline missions cause errors in console and the questor get stucked.

Is the only error i have find with the last version.

**Comment by ISeeDEDPpl** [comment]
can you paste the console output and/or the innerspace log (if your are seeing exceptions)?

**Comment by Chryst** [comment]
yes i see expection, it said error at x line with a big white spam in console. i was with 3 low sec storyline in journal at station, and it have write unable to find xml file, but the file there was in the folder, when it come out again i copy paste it, thx

**Comment by Chryst** [comment]
That is happend when i have log with 3 low sec stoyline in diary

19:34:49 [CharSchedule] No start time specified. Starting now.
19:34:49 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:49 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:49 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:49 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:49 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] RunTime Attribute no longer supported: use StartTime and StopTime
19:34:50 [CharSchedule] No start time specified. Starting now.
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No start time specified. Starting now.
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No start time specified. Starting now.
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No start time specified. Starting now.
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:50 [CharSchedule] No start time specified. Starting now.
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] RunTime Attribute no longer supported: use StartTime and StopTime
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No start time specified. Starting now.
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [CharSchedule] No stop time specified: Assuming 4 hours
19:34:51 [Startup] User: XXXACCOUNTXXXX Name: XXXCHARXXXX
19:34:51 [Startup] Forcing Schedule 1 because none of the schedules started within 2 hours
19:34:51 [Startup] Schedule 1: Start1: 09/10/2012 00:00:00 Stop1: 09/10/2012 01:30:00
19:34:51 [Startup] Start Time: 09/10/2012 19:34:51 - Stop Time: 10/10/2012 01:30:00
19:34:52 [Startup] Already passed start time. Starting in 15 seconds.
19:35:09 [Startup] Login account [XXXXXX]
19:35:09 [Startup] Waiting for Character Selection Screen
19:35:49 [Startup] Activating character [XXXCHARXXXX]
19:36:29 [Startup] We've successfully logged in
19:36:33 [Questor] Schedule: setup correctly: stoptime is [01:30]
19:36:33 [Questor] EVE instance: totalMegaBytesOfMemoryUsed - 632 MB
19:36:33 [Settings] Loading Settings from [C:\InnerSpace.NET Programs\XXXCHARXXXX.xml]
19:36:33 [Settings] MissionsPath is: [C:\InnerSpace.NET Programs\Caldari L4 Missions ESPV]
19:36:33 [Settings] Testing: LavishScript Version 1.96
19:36:33 [Settings] Found only one agent in your character XML: MultiAgentSupport is [False]
19:36:33 [Settings] Loading Mission Blacklist
19:36:33 [Settings] Mission Blacklist now has [10] entries
19:36:33 [Settings] Loading Mission Greylist
19:36:33 [Settings] Mission Greylist now has [10] entries
19:36:33 [Settings] Loading Faction Blacklist
19:36:33 [Settings] Missions from the faction [Le fa tutte] will be declined
19:36:33 [Settings] Faction Blacklist now has [1] entries
19:36:34 [Cache.CurrentAgent] [ XXXAGENTXXX ] AgentID [ XXXAGENTXXX ]
19:36:34 [Logging][ XXXAGENTXXX ] AgentID [ XXXAGENTXXX ]
19:36:36 [Questor] Checking: Current time [10/09/2012 19:36:36] StopTimeSpecified [True] StopTime [ 10/10/2012 01:30:00] ManualStopTime = 10/10/2012 XX:XX:49
19:36:36 [Questor] Start Mission Behavior
19:36:37 [Statistics: WriteMissionStatistics] It is unlikely a mission has been run yet this session... No Mission log needs to be written.
19:36:38 [CombatMissionsBehavior] Random start delay of [8] seconds
19:36:43 [Statistics: WriteSessionLogStarting] Writing session data to [ C:\InnerSpace.NET Programs\log\XXXCHARXXXX\XXXCHARXXXX.Sessions.log ]
19:36:48 [CombatMissionsBehavior.TravelToAgentsStation] Destination: [XXXSTAXXX XXXSTAXXX]
19:36:48 [TravelerDestination.StationDestination] Destination set to [XXXSTAXXX XXXSTAXXX]
19:36:50 [TravelerDestination.StationDestination] Arrived in station
19:36:50 [CombatMissionsBehavior.TravelToAgentsStation] Arrived at destination
19:36:52 [DroneStats] We do not use drones in this type of ship, skipping dronestats
19:36:52 [Statistics: AmmoConsumptionStats] Opening Cargohold of active ship: waiting [3sec]
19:36:55 [AgentInteraction] Start Conversation [Complete Mission]
19:36:55 [AgentInteraction] Waiting for conversation
19:36:58 [AgentInteraction] Replying to agent
19:37:04 [Questor] Running Innerspace command: windowtaskbar on XXXCHARXXXX
19:37:05 [AgentInteraction] Saying [Complete Mission]
19:37:05 [AgentInteraction] Closing conversation
19:37:07 [Cleanup] This window indicates an error completing a mission: [1] errors already we will stop questor and halt restarting when we reach 3
19:37:12 [AgentInteraction] Attempting to close Agent Window
19:37:14 [AgentInteraction] Done
19:37:15 [CombatMissionsBehavior] UnloadLoot: Begin
19:37:15 [CombatMissionsBehavior.Unloadloot] We are on mission
19:37:16 [CombatMissionsBehavior] Random start delay of [5] seconds
19:37:22 [Questor] EVE instance: totalMegaBytesOfMemoryUsed - 676 MB
19:37:23 [Storyline] Currently have [3] availible storyline missions
19:37:23 [Storyline] Currently have [3] storyline missions questor knows how to do
19:37:23 [Storyline] Currently have [3] storyline missions questor knows how to do and are not blacklisted
19:37:23 [CombatMissionsBehavior] Storyline detected, doing storyline.
19:37:24 [Storyline] Currently have [3] availible storyline missions
19:37:24 [Storyline] Currently have [3] storyline missions questor knows how to do
19:37:24 [Storyline] Currently have [3] storyline missions questor knows how to do and are not blacklisted
19:37:24 [Storyline] Going to do [Shipyard Theft] for agent [Ushokera Pisto] AgentID[3017112]
19:37:26 [AgentInteraction] Waiting for conversation
19:37:29 [AgentInteraction] Checking ammo type
19:37:30 [AgentInteraction] [Ushokera Pisto] standing toward me is [0,00], minAgentGreyListStandings: [-1,7]
19:37:30 [AgentInteraction] Mission enemy faction: Blood Raiders
19:37:30 [GenericCombatStoryline] Mission offer is in a Low Security System
19:37:30 [GenericCombatStoryline] Something went wrong, blacklist this agent [Low security systems]
19:37:31 [Storyline] BlacklistAgent: The agent that provided us with this storyline mission has been added to the session blacklist
19:37:32 [CombatMissionsBehavior.TravelToAgentsStation] Destination: [XXXSTAXXX XXXSTAXXX]
19:37:32 [TravelerDestination.StationDestination] Destination set to [XXXSTAXXX XXXSTAXXX]
19:37:34 [TravelerDestination.StationDestination] Arrived in station
19:37:34 [CombatMissionsBehavior.TravelToAgentsStation] Arrived at destination
19:37:35 [AgentInteraction] Start Conversation [Complete Mission]
19:37:35 [AgentInteraction] Waiting for conversation
19:37:38 [AgentInteraction] Replying to agent
19:37:45 [AgentInteraction] Saying [Complete Mission]
19:37:45 [AgentInteraction] Closing conversation
19:37:50 [Cleanup] This window indicates an error completing a mission: [2] errors already we will stop questor and halt restarting when we reach 3
19:37:54 [AgentInteraction] Attempting to close Agent Window
19:37:56 [AgentInteraction] Done
19:37:57 [CombatMissionsBehavior] UnloadLoot: Begin
19:37:57 [CombatMissionsBehavior.Unloadloot] We are on mission
19:37:58 [CombatMissionsBehavior] Random start delay of [8] seconds
19:38:08 [Questor] EVE instance: totalMegaBytesOfMemoryUsed - 576 MB
19:38:09 [Storyline] Currently have [2] availible storyline missions
19:38:09 [Storyline] Currently have [2] storyline missions questor knows how to do
19:38:09 [Storyline] Currently have [2] storyline missions questor knows how to do and are not blacklisted
19:38:09 [CombatMissionsBehavior] Storyline detected, doing storyline.
19:38:10 [Storyline] Currently have [2] availible storyline missions
19:38:10 [Storyline] Currently have [2] storyline missions questor knows how to do
19:38:10 [Storyline] Currently have [2] storyline missions questor knows how to do and are not blacklisted
19:38:10 [Storyline] Going to do [Forgotten Outpost] for agent [Rantoh Funkaki] AgentID[3018696]
19:38:11 [AgentInteraction] Waiting for conversation
19:38:15 [AgentInteraction] Checking ammo type
19:38:16 [AgentInteraction] [Rantoh Funkaki] standing toward me is [0,00], minAgentGreyListStandings: [-1,7]
19:38:16 [AgentInteraction] Mission enemy faction: Guristas Pirates
19:38:16 [GenericCombatStoryline] Mission offer is in a Low Security System
19:38:16 [GenericCombatStoryline] Something went wrong, blacklist this agent [Low security systems]
19:38:17 [Storyline] BlacklistAgent: The agent that provided us with this storyline mission has been added to the session blacklist
19:38:18 [CombatMissionsBehavior.TravelToAgentsStation] Destination: [XXXSTAXXX XXXSTAXXX]
19:38:18 [TravelerDestination.StationDestination] Destination set to [XXXSTAXXX XXXSTAXXX]
19:38:20 [TravelerDestination.StationDestination] Arrived in station
19:38:20 [CombatMissionsBehavior.TravelToAgentsStation] Arrived at destination
19:38:21 [AgentInteraction] Start Conversation [Complete Mission]
19:38:21 [AgentInteraction] Waiting for conversation
19:38:24 [AgentInteraction] Replying to agent
19:38:31 [AgentInteraction] Saying [Complete Mission]
19:38:31 [AgentInteraction] Closing conversation
19:38:33 [Cleanup] This window indicates an error completing a mission: [3] errors already we will stop questor and halt restarting when we reach 3
19:38:41 [AgentInteraction] Attempting to close Agent Window
19:38:43 [AgentInteraction] Done
19:38:44 [CombatMissionsBehavior] UnloadLoot: Begin
19:38:44 [CombatMissionsBehavior.Unloadloot] We are on mission
19:38:45 [CombatMissionsBehavior] Random start delay of [3] seconds
19:38:49 [Questor] EVE instance: totalMegaBytesOfMemoryUsed - 530 MB
19:38:50 [Storyline] Currently have [1] availible storyline missions
19:38:50 [Storyline] Currently have [1] storyline missions questor knows how to do
19:38:50 [Storyline] Currently have [1] storyline missions questor knows how to do and are not blacklisted
19:38:50 [CombatMissionsBehavior] Storyline detected, doing storyline.
19:38:51 [Storyline] Currently have [1] availible storyline missions
19:38:51 [Storyline] Currently have [1] storyline missions questor knows how to do
19:38:51 [Storyline] Currently have [1] storyline missions questor knows how to do and are not blacklisted
19:38:51 [Storyline] Going to do [Forgotten Outpost] for agent [Avas Hintinen] AgentID[3018697]
19:38:52 [AgentInteraction] Waiting for conversation
19:38:56 [AgentInteraction] Checking ammo type
19:38:57 [AgentInteraction] [Avas Hintinen] standing toward me is [0,00], minAgentGreyListStandings: [-1,7]
19:38:57 [AgentInteraction] Mission enemy faction: Guristas Pirates
19:38:57 [GenericCombatStoryline] Mission offer is in a Low Security System
19:38:57 [GenericCombatStoryline] Something went wrong, blacklist this agent [Low security systems]
19:38:58 [Storyline] BlacklistAgent: The agent that provided us with this storyline mission has been added to the session blacklist
19:38:59 [CombatMissionsBehavior.TravelToAgentsStation] Destination: [XXXSTAXXX XXXSTAXXX]
19:38:59 [TravelerDestination.StationDestination] Destination set to [XXXSTAXXX XXXSTAXXX]
19:39:01 [TravelerDestination.StationDestination] Arrived in station
19:39:01 [CombatMissionsBehavior.TravelToAgentsStation] Arrived at destination
19:39:02 [AgentInteraction] Start Conversation [Complete Mission]
19:39:02 [AgentInteraction] Waiting for conversation
19:39:05 [AgentInteraction] Replying to agent
19:39:12 [AgentInteraction] Saying [Complete Mission]
19:39:12 [AgentInteraction] Closing conversation
19:39:17 [Cleanup] This window indicates an error completing a mission: [4] errors already we will stop questor and halt restarting when we reach 3
19:39:17 [Cleanup] ERROR: Mission XML is missing for [Forgotten Outpost] and we we unable to complete the mission after 3 tries! - ERROR!
19:40:39 [Questor] Wallet Balance Has Not Changed in [ 2 ] minutes.
19:41:39 [Questor] Wallet Balance Has Not Changed in [ 3 ] minutes.
19:42:40 [Questor] Wallet Balance Has Not Changed in [ 4 ] minutes.

**Comment by Chryst** [comment]
19:39:17 [Cleanup] ERROR: Mission XML is missing for [Forgotten Outpost] and we we unable to complete the mission after 3 tries! - ERROR!

with that was come out some white exception in console, said error at line xx, but i haven't copy it

**Comment by ISeeDEDPpl** [comment]
ok this is a combination of bugs unfortunately... the 1st is that MissionCompletionErrors was not being reset to 0 when we declined a mission, so it would rise by 1 for every mission you declined, that is now fixed. \

https://github.com/ISeeDEDPpl/Questor/commit/0a018e73c04a0fe41dd41e5da83407825b1cdc5c

The 2nd part is the exception you are seeing in the innerspace console that seems unrelated. It is a problem for sure and needs to be fixed but I cant see where it would be occurring. I will need the innerspace log from a toon where that is happening.  "log NameofMyInnerspaceLog.log"
